packages/server/modules/core/domain/commits/types.ts

```diff
@@ -15,3 +21,34 @@ export type CommitWithStreamBranchMetadata = Commit & {
 }
 
 export type CommitBranch = Branch & { commitId: string }
+
+export type LegacyUserCommit = {
+  id: CommitRecord['id']
+  message: CommitRecord['message']
+  referencedObject: CommitRecord['referencedObject']
+  sourceApplication: CommitRecord['sourceApplication']
+  totalChildrenCount: CommitRecord['totalChildrenCount']
+  parents: CommitRecord['parents']
+  createdAt: CommitRecord['createdAt']
+  branchName: BranchRecord['name']
+  streamId: StreamCommitRecord['streamId']
+  streamName: StreamRecord['name']
+  authorName: UserRecord['name']
+  authorId: UserRecord['id']
+  authorAvatar: UserRecord['avatar']
+}
+
+export type LegacyStreamCommit = {
```

Comment:
Needed to define this one in advance for getStreamCommits to appease TS. The actual fn that uses this will come in subsequent PRs.